src/slack/slack.js

```diff
@@ -693,15 +694,27 @@ export class Slack {
     }
 
     // office-personal-?-note
-    if (action === 'note') {
+    if (actionName === 'note') {
       await submitOrder(actionValue === 'note-yes' ? MESSAGES.office.personal.note : null)
       return true
     }
 
     // ?-company
-    if (action === 'company') {
+    if (actionName === 'company') {
       order.isCompany = true
 
+      await updateMessage({
```

Comment:
I just noticed this logic is not scoped to the test variant. this means when we merge it, it would ask for spinoff on both VL and Wincent workspaces, which is undesired. so what needs to be done:
- branch the logic here based on `this.variant` - for `wincent`, run the original code (`submitOrder`), for others run this new code
- for `actionName === 'spinoff'` I wouldn't do anything, just put a comment above that it is unreachable for wincent

in other code it should be OK.